src/main/java/entity_gateway/EventEntityManager.java

```diff
@@ -43,4 +43,15 @@ public boolean addEvent(CalendarEventModel eventData) {
         }
         return eventReaderList;
     }
+    @Override
+    public boolean update(Long eventId, LocalDateTime startTime, LocalDateTime endTime) {
+        for(Event event: this.eventList){
+            if (event.getEventName().equals(eventId)){
```

Comment:
Shouldn't we check for if `event.getEventId().equals(eventId)` instead of using event name?